tests/framework/jailer.py

```diff
@@ -110,6 +113,10 @@ def construct_param_list(self):
             jailer_param_list.append('--daemonize')
         if self.new_pid_ns:
             jailer_param_list.append('--new-pid-ns')
+        if self.cgroup_ver:
```

Comment:
Should we test the jailer with v1 and then with `cgroup_ver` set to None? It seems that we're only exercising the v2 path.

Reply:
Yes, makes sense to test with cgroup_ver set to 1 and 2 and also without the parameter set. I'll leave the default to None and add another test.